search/linear_search.cpp

```diff
@@ -32,7 +35,7 @@ int main() {
     int *array = new int[size];
     int key;
 
-    // Input array
+    /*User gets to input array elements*/
     std::cout << "\nEnter the Array of " << size << " numbers : ";
```

Comment:
There should be a check for the user entering a negative number or a very large number as "size" of the array of numbers.